DQMOffline/Trigger/python/HiggsMonitoring_cff.py

```diff
@@ -3,11 +3,13 @@
 from DQMOffline.Trigger.VBFMETMonitor_cff import *
 from DQMOffline.Trigger.MssmHbbBtagTriggerMonitor_cff import *
 from DQMOffline.Trigger.MssmHbbMonitoring_cff import *
+from DQMServices.Core.DQMEDHarvester import DQMEDHarvester
 
 higgsMonitorHLT = cms.Sequence(
 
     higgsinvHLTJetMETmonitoring
-    mssmHbbBtagTriggerMonitor +
-    mssmHbbMonitorHLT +
+  + mssmHbbBtagTriggerMonitor 
+  + mssmHbbMonitorHLT 
+  + higgsHLTDiphotonMonitoring
```

Comment:
thanks for spotting this, I missed the module DQMOffline.Trigger.PhotonMonitor_cff of the diphoton group now I have included it